docs/dice_verification.md

```diff
@@ -1,40 +1,137 @@
 ## Verifying dice seed generation 
 
 It is possible to do a 'dry run' to verify that seed generation has not been tempered.
-This will ensure that the derivation algorithm has not been tempered and is still same as well known algorithm used bu coldcard and ian coleman bip39 webpage.
+This will ensure that the derivation algorithm has not been tempered and is still the same as the well known algorithm used by coldcard and Ian Coleman bip39 webpage.
```

Comment:
Mention that it is checking to Seed Tool and Ian Coleman. Coldcard can be removed as a reference because it requires the person to purchase a coldcard which they may not have. Checking against the 2 separate websites seems sufficient to me.